packages/react-components/react-color-picker-preview/stories/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "devDependencies": {
-    "@ctrl/tinycolor": "3.3.4",
+    "@ctrl/tinycolor": "^3.3.4",
```

Comment:
FYI @ValentinaKozlova this shouldn't be here. we use single version policy resources that are present within monorepo root for non production purposes 

```suggestion
```